### DISCUSSION TITLE
[Errno 2] No such file or directory message

### DISCUSSION THREAD

**Discussion by aomatl** [nianeyna/ao3downloader#59] [Q&A] [answered]
Hi, this is my first time here so please let me know if I've missed out on anything! ^^;; I downloaded the latest version for ao3downloader (as of Oct. 5) and I currently have the Python 3.10.5 version.

I tried to 'get all work links from an ao3 listing (saves links only)' using my bookmarks link and received the error message `[Errno 2] No such file or directory: 'downloads\\links_10052022233056.txt'` after reaching the last page. I checked my Downloads folder and there doesn't seem to be a .txt with the file name (the ao3downloader folder is in my Downloads folder as well, if that helps!). Here's the end portion if needed!:

`finished getting page 39. starting page 40`
`[Errno 2] No such file or directory: 'downloads\\links_10052022233056.txt'`
`'d' to display the menu again`
`please enter your choice, or 'q' to quit:`

I'm not sure where to go from here. Asking for advice ahh, sorry for the trouble and thank you!

**Comment by nianeyna** [comment, marked as answer]
Oh god thank you so much for reporting this lol. That was definitely a bug. I've now fixed it so if you grab the latest version of the code you should be good to go. I'm so sorry about that!

**Reply by aomatl** [reply]
No worries, it works perfectly now! Thank you so much!! ^^
